### PR TITLE
feature(next-migration): next conference

### PR DESCRIPTION
**Pull request by siradji**

### CODE REVIEW

**Comment by gomandev** [comment]
@Bbanez @boris-becomes looking forward to your review. :)

File: next/conference/components/home-page/Speakers.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                      { speaker.name }
+                    </div>
+                    <div
+                      className="text-xs leading-none tracking-[-0.05em] font-medium text-appGray-500 lg:text-2xl lg:leading-none"
+                    >
+                      { speaker.role }
+                    </div>
+                  </div>
+                </div>
+                <ContentManager
```
Comment (boris-becomes) [inline next/conference/components/home-page/Speakers.tsx]: @siradji Missing styles for the "Speakers" section ( font size mainly )
267946131-db23de48-4467-434d-a470-66e0ca1788a9

File: next/conference/components/home-page/Sponsors.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                'flex justify-center px-1 py-[7px] rounded-[38px] text-xs leading-none tracking-[-0.04em] transition-colors duration-300 lg:py-[18px] lg:text-2xl lg:leading-none',
+                {
+                  'text-white bg-appAccent font-semibold': activeTier === tier,
+                  'text-appGray-600 font-medium': activeTier !== tier,
+                }
+              )}
+              onClick={() => setActiveTier(tier)}
+              key={tier}
+            >
+              {tier}
```
Comment (boris-becomes) [inline next/conference/components/home-page/Sponsors.tsx:60]: @siradji Tier labels should be capitalized like on https://conference-nuxt.thebcms.com/.
Check if filtering/swiping is working after the update.

Comment (siradji) [reply next/conference/components/home-page/Sponsors.tsx:60]: @boris-becomes , all fixed. please review

**Comment by boris-becomes** [comment]
@siradji Please also check the eslint/prettier packages we have used for Nuxt starters and install them and update the "lint" script command which you can run before pushing the code ( `npm run lint` ). It will give you errors for wrong spacings, missing types, etc. so please fix those as well. Check the .prettierrc file as well and compare to Nuxt. Thanks!

**Comment by siradji** [comment]
@boris-becomes , installed the proper linting package

**Comment by boris-becomes** [comment]
Hi @siradji , one more step is left: to update the "lint" script from package.json - `"lint": "eslint . --ext .ts,.tsx"`. And inside the `.eslintrc.json` file you should extend the `"plugin:prettier/recommended"` plugin. Now when you run `npm run lint` you will see a lot of spacing and similar errors which you can fix more-less by just formatting the file ( it should pick prettier formatting rules ).

**Comment by siradji** [comment]
@boris-becomes  done

**Review by boris-becomes — APPROVED** [review]

**Comment by boris-becomes** [comment]
@siradji I still get formatting errors in ~15 files, you don't? Also, I just noticed the title "Insightful Ink" is being used in PageWrapper component and og-head composable, can you update that to "CONference"? Just minor issues, but let's push it without these errors. Thank you for your work!

**Comment by siradji** [comment]
> @siradji I still get formatting errors in ~15 files, you don't? Also, I just noticed the title "Insightful Ink" is being used in PageWrapper component and og-head composable, can you update that to "CONference"? Just minor issues, but let's push it without these errors. Thank you for your work!

@boris-becomes, i do not get any issues with linter. Could you perhaps pull and update the project? The formatting issue were all fixed.

**Pull request merged**